src/app/core/services/personal-cards.service.spec.ts

```
@@ -189,16 +191,47 @@ describe('PersonalCardsService', () => {
-    expenseAggregationService.delete.and.returnValue(of(deletePersonalCardRes));
+  describe('deleteAccount()', () => {
+    it('should delete a personal card when using public API', fakeAsync(() => {
+      personalCardsService.usePlatformApi = false;
+      expenseAggregationService.delete.and.returnValue(of(deletePersonalCardRes));
+      spyOn(personalCardsService, 'deleteAccountPlatform');
+
+      const accountId = 'bacc0By33NqhnS';
 
-    const accountId = 'bacc0By33NqhnS';
+      let response: PersonalCard;
+      personalCardsService.deleteAccount(accountId).subscribe((res) => (response = res));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)